ui/app/templates/vault/cluster/init.hbs

```diff
@@ -95,15 +95,14 @@
             </div>
           {{/if}}
           <DownloadButton
-            class="button is-ghost"
+            @color="tertiary"
```

Comment:
<img width="749" alt="Screenshot 2023-09-28 at 4 33 21 PM" src="https://github.com/hashicorp/vault/assets/68122737/9914c656-3e31-4b01-9481-d8139233e640">

Reply:
Most download actions have the icon trailing, so opted to make that a default instead of _another_ arg to pass in

Reply:
Changed this! HDS defaults to 'leading' so keeping that functionality to align with other products